source/macos/AdaptiveCards/AdaptiveCards/Views/ACRMultilineInputTextView.swift

```diff
@@ -136,6 +169,14 @@ class ACRMultilineInputTextView: NSView, NSTextViewDelegate {
             textView.backgroundColor = isMouseInView ? inputConfig.highlightedColor : inputConfig.backgroundColor
         }
     }
+    
+    private func setupInputElementProperties(inputElement: ACSBaseInputElement?) {
+        if config.supportsSchemeV1_3 {
+            self.labelString = inputElement?.getLabel()
+            self.errorMessage = inputElement?.getErrorMessage()
```

Comment:
```suggestion
        guard config.supportsSchemeV1_3 else  { return }
        labelString = inputElement?.getLabel()
        errorMessage = inputElement?.getErrorMessage()
```

`guard...else` can be used instead.
And `self` can be ignored.